server/utils/nunjucksSetup.ts

```diff
@@ -63,6 +63,54 @@ export default function nunjucksSetup(app: express.Express): void {
     return encodeURI(`${date.getDate()}/${date.getMonth() + 1}/${date.getFullYear()}`)
   })
 
+  njkEnv.addFilter('findError', (array, formFieldId) => {
+    if (!array) return null
+    // @ts-ignore
+    const item = array.find((error) => error.href === `#${formFieldId}`)
+    if (item) {
+      return {
+        text: item.text,
+      }
+    }
+    return null
+  })
+
+  njkEnv.addFilter('showDefault', (value, specifiedText) => {
+    if (value === 0) return value
+
+    return value || specifiedText || '--'
+  })
+
+  njkEnv.addFilter('addDefaultSelectedVale', (items, text, show) => {
```

Comment:
is this supposed to be `addDefaultSelectedValue` (as in "value") or am I missing something?